src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/UsePropertiesWhereAppropriateTests.cs

```diff
@@ -345,6 +376,32 @@ End Class
             GetCA1024BasicResultAt(17, 24, "GetFileNameProtected"));
         }
 
+        [Fact, WorkItem(1432, "https://github.com/dotnet/roslyn-analyzers/issues/1432")]
+        public void VisualBasic_CA1024NoDiagnosticCases_Internal()
+        {
+            VerifyBasic(@"
+Public Class Class1
+	Private fileName As String
```

Comment:
>	 [](start = 0, length = 1)

This file mixes tabs and spaces. This is horrifying.

Reply:
Fixed.